### PR TITLE
feat: [curve/toos-v2]: add list client #2037

### PR DESCRIPTION
**Pull request by tsonglew**
### What problem does this PR solve?

Issue Number:  #2037

Problem Summary:

add subcommand client in curve bs list

What's Changed:

image

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by cw123** [comment]
add the `list client`command to the tools-v2/README.md

**Comment by tsonglew** [comment]
> add the `list client`command to the tools-v2/README.md

updated

**Comment by tsonglew** [comment]
> It needs to be added that `mdsaddr ` reads from the configuration file by default.
> 
> image
> 
> image

it looks good with my config files in the default dir

image

could you please leave a copy of your curve.yaml ?

**Comment by ilixiaocui** [comment]
> > It needs to be added that `mdsaddr ` reads from the configuration file by default.
> > image
> > image
> 
> it looks good with my config files in the default dir
> 
> image
> 
> could you please leave a copy of your curve.yaml ?

Sorry, it’s ok now, I just wrote the name of the configuration file as `client.yaml`.

LGTM!

**Review by ilixiaocui — APPROVED** [review]

**Comment by ilixiaocui** [comment]
We are looking for developers, could you please add curve's WeChat？
image

File: tools-v2/pkg/cli/command/curvebs/list/client/client.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+	timeout := config.GetFlagDuration(pCmd.Cmd, config.RPCTIMEOUT)
+	retrytimes := config.GetFlagInt32(pCmd.Cmd, config.RPCRETRYTIMES)
+
+	rpc := &ListClientRpc{
+		Request: &nameserver2.ListClientRequest{},
+		Info:    basecmd.NewRpc(mdsAddrs, timeout, retrytimes, "ListClient"),
+	}
+	pCmd.Rpc = append(pCmd.Rpc, rpc)
+
+	header := []string{cobrautil.ROW_ADDR}
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/list/client/client.go:106]: Separate ip and port, is it better to sort by ip? Like this:
```
+------------------+--------+
|       IP         |  PORT  |
+------------------+--------+
| 10.182.2.48      |  9001  |
+                  +--------+
|                  |  9002  |
+------------------+--------+
```

Comment (tsonglew) [reply tools-v2/pkg/cli/command/curvebs/list/client/client.go]: updated
image

**Review by Cyber-SiKu — APPROVED** [review]
LGTM!

**Comment by ilixiaocui** [comment]
recheck

**Comment by cw123** [comment]
It is best to combine multiple git commits into one.

image

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by tsonglew** [comment]
> It is best to combine multiple git commits into one.
> 
> image

@cw123 PTAL

**Comment by Cyber-SiKu** [comment]
recheck

**Pull request merged**